File: CurriculumRepository.CORE/Data/Models/Activity/LaBM.cs
using CurriculumRepository.CORE.Entities;
using System.Collections.Generic;

namespace CurriculumRepository.CORE.Data.Models.Activity
{
    public class LaBM
    {
        public LaBM()
        {
            StrategyMethods = new List<StrategyMethodBM>();
            LaTeachingAidUser = new List<TeachingAid>();
            LaTeachingAidTeacher = new List<TeachingAid>();
        }
        public int OrdinalNumber { get; set; }
        public int? PerformanceId { get; set; }
        public int LatypeId { get; set; }
        public bool DigitalTechnology { get; set; }
        public string Laname { get; set; }
        public int LadurationMinute { get; set; }
        public string Ladescription { get; set; }
        public List<StrategyMethodBM> StrategyMethods { get; set; }
        public int CooperationId { get; set; }
        public List<TeachingAid> LaTeachingAidUser { get; set; }
        public List<TeachingAid> LaTeachingAidTeacher { get; set; }
    }
}

File: CurriculumRepository.CLIENT/client-app/src/app/models/activity/LaBM.ts
import { StrategyMethodBM } from '../StrategyMethodBM';
import { TeachingAid } from '../TeachingAid';

export class LaBM {
    OrdinalNumber: number;
    PerformanceId: number;
    LatypeId: number;
    DigitalTechnology: boolean;
    Laname: string;
    LadurationMinute: number;
    Ladescription: string;
    StrategyMethods: StrategyMethodBM[];
    CooperationId: number;
    LaTeachingAidUser: TeachingAid[];
    LaTeachingAidTeacher: TeachingAid[];
}